tests/helpers/attributes.js

```diff
@@ -0,0 +1,17 @@
+import { test } from 'ember-qunit';
+import Ember from 'ember';
+import 'code-corps-ember/tests/helpers/has-attributes';
+
+const { get } = Ember;
+
+// modified from gist source for relationships
+// source: https://gist.github.com/he9qi/b6354a81a0672dc63294
+export function testForAttributes(name, expectedAttributes) {
+  test(`should have attributes [${expectedAttributes.join(', ')}]`, function(assert) {
+    assert.expect(1);
+    let Model = this.store().modelFor(name);
+    let actualAttributes = get(Model, 'attributes');
+
+    assert.hasAttributes(actualAttributes, expectedAttributes);
+  });
+}
```

Comment:
We can move this and `tests/helpers/relationhips` into a common `test/helpers/model` and then get rid of `has-attributes`, since that part can be mimicked from within this helper, without the need to monkeypatch the `QUnit` prototype.